core/src/test/java/nl/nn/adapterframework/pipes/HashPipeTest.java

```diff
@@ -47,6 +47,15 @@ public void wrongAlgorithm() throws ConfigurationException, PipeStartException,
 		pipe.configure();
 	}
 
+	@Test
+	public void wrongBinaryToTextEncoding() throws ConfigurationException, PipeStartException, IOException, PipeRunException {
+		exception.expect(ConfigurationException.class);
```

Comment:
Ik zou ook graag hierbij een test zien dat er een duidelijke foutmelding is.

Reply:
Gedaan, ook voor de wrongAlgorithm-test.